src/data/markdown/translated-guides/en/02 Using k6/06 Test life cycle.md

```diff
@@ -41,33 +62,45 @@ export default function () {
 
 </CodeGroup>
 
-_"Why not just run my script normally, from top to bottom"_, you might ask - the answer is: we
-do, but code **inside** and **outside** your `default` function can do different things.
+Why not just run my script normally, from top to bottom?
 
-Code inside `default` is called "VU code", and is run over and over for as long as the test is
-running. Code outside of it is called "init code", and is run only once per VU.
+This is a good question, and the answer is: we do.
+But code *inside* and *outside* your `default` function do different things.
 
-VU code can make HTTP requests, emit metrics, and generally do everything you'd expect a load
-test to do - with a few important exceptions: you can't load anything from your local filesystem,
-or import any other modules. This all has to be done from the init code.
+Code inside `default` is called *VU code*.
+VU code runs over and over through the test duration.
+Besides setup and teardown, code outside of `default` is  *init code*.
+It runs only once per VU.
 
-We have two reasons for this. The first is, of course: performance.
+VU code can make HTTP requests, emit metrics, and generally do everything you'd expect a load test to do.
+There are a few important exceptions. VU code:
+* *Does not* load files from your local filesystem,
+* *Does not* import any other modules. 
 
-If you read a file from disk on every single script iteration, it'd be needlessly slow; even
-if you cache the contents of the file and any imported modules, it'd mean the _first run_ of the
-script would be much slower than all the others. Worse yet, if you have a script that imports
-or loads things based on things that can only be known at runtime, you'd get slow iterations
-thrown in every time you load something new.
+Instead of VU code, init code does these jobs.
 
-But there's another, more interesting reason. By forcing all imports and file reads into the
-init context, we make an important design goal possible; we want to support three different
-execution modes without the need for you to modify your scripts; local, cloud and clustered
-execution. In the case of cloud and clustered execution we know which files will be needed, so
-we distribute only those files. We know which modules will be imported, so we can bundle them
-up from the get-go. And, tying into the performance point above, the other nodes don't even
-need writable filesystems - everything can be kept in-memory.
+Why separate init and VU code? There are two chief reasons:
+* Better performance
+* Better design
 
-As an added bonus, you can use this to reuse data between iterations (but only for the same VU):
+### Performance is faster and more accurate
```

Comment:
I suggest removing both sections. Perhaps a short explanation with the bullet points. 

If we want to expand the topic, we could create an additional section on other page.

Reply:
Now, I've removed the H3s, and made that section its own H2.

What do you think? It seems like it works as its own section. I believe it's good to split content into digestible sections, making it a little easier for the reader to scan, and avoiding the appearance of the dreaded "wall of text".